### PR TITLE
compute: Allow setting the hostname when creating a server

### PR DESCRIPTION
**Pull request by till**
Related: #2967

### CODE REVIEW

File: openstack/compute/v2/servers/requests.go [hunk 1]
```diff
@@ -241,6 +241,12 @@ type CreateOpts struct {
 	// Tags allows a server to be tagged with single-word metadata.
 	// Requires microversion 2.52 or later.
 	Tags []string `json:"tags,omitempty"`
+
+	// (Available from 2.9.0) Hostname specifies the hostname to configure for the
```
Comment (mandre) [inline openstack/compute/v2/servers/requests.go:245]: Nit, a small dot slipped in.
```suggestion
	// (Available from 2.90) Hostname specifies the hostname to configure for the
```

Comment (till) [reply openstack/compute/v2/servers/requests.go]: Thank you @mandre! Fixed!

**Review by mandre — APPROVED** [review]

**Comment by till** [comment]
@mandre can you tell why it fails? I can't find anything in the log output. But let me know if I broke anything.

**Comment by mandre** [comment]
> @mandre can you tell why it fails? I can't find anything in the log output. But let me know if I broke anything.

Second time it hit a timeout when rebuilding the server. Looking at the periodic jobs, this looks like the compute job has been failing with the same error on zed for quite some time. Definitely something to investigate, but also clearly nothing related to your patch. Let's merge it.

**Comment by till** [comment]
Thank you!

**Pull request merged**